vdm-stub-generator/build.gradle

```diff
@@ -11,14 +12,73 @@ dependencies {
     runtimeOnly "com.anaplan.engineering:vdm-animation-overture:$version"
 }
 
-
 gradlePlugin {
     plugins {
         vdmStubGenerator {
-            id = "vdm-stub-generator"
+            id = "${group}.vdm-stub-generator"
+            displayName = 'VDM stub generator gradle plugin'
+            description = 'A Gradle plugin to generate stub classes from the structure of a VDM specification'
             implementationClass = "com.anaplan.engineering.vdmstubgenerator.VdmStubGenerationPlugin"
         }
     }
 }
 
+pluginBundle {
+    website = 'https://github.com/anaplan-engineering/azuki'
+    vcsUrl = 'https://github.com/anaplan-engineering/azuki'
+    tags = ['VDM']
+}
+
+java {
+    withSourcesJar()
+    withJavadocJar()
+}
+
+publishing {
+    afterEvaluate {
+        publications {
+            withType(MavenPublication) {
+                groupId = group
+                pom {
+                    url = 'https://github.com/anaplan-engineering/azuki'
+                    name = 'VDM stub generator gradle plugin'
+                    description = 'A Gradle plugin to generate stub classes from the structure of a VDM specification'
```

Comment:
Would it be worth storing these in properties/variables instead of repeating them here and in the `gradlePlugin` block?

Reply:
I just put in something minimal, but really they should probably be different